packages/rfw/test_coverage/bin/test_coverage.dart

```diff
@@ -0,0 +1,104 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:lcov_parser/lcov_parser.dart' as lcov;
+
+// Please update these targets when you update this package.
+// Please ensure that test coverage continues to be 100%.
+const int targetLines = 2127;
+const String targetPercent = '100';
+const String lastUpdate = '2021-08-30';
+
+Future<void> main(List<String> arguments) async {
+  // This script is mentioned in the README.md file.
+
+  if (Platform.environment['CHANNEL'] == 'stable') {
+    // For now these are disabled because this package has never been supported
+    // on the stable channel and requires newer language features that have not
+    // yet shipped to a stable build. -Hixie, 2021-08-30
```

Comment:
Maybe give a version number where it'll be supported, so it'll be clear when this can be removed?